### ISSUE TITLE
error adding searchbox url to heroku 

### ISSUE THREAD

**Issue by WilfredKesh** [ankane/searchkick#703]
I ran
heroku config:set ELASTICSEARCH_URL=`heroku config:get SEARCHBOX_URL`
and am getting a "config:get is invalid, must be in the format Foo=bar"  error in the command line

**Comment by ACPK** [comment]
@WilfredKesh Have you added an Elastic add-on yet?

**Comment by WilfredKesh** [comment]
Yea I added the searchbox starter
heroku addons:create searchbox:starter

**Comment by WilfredKesh** [comment]
I checked the SEARCHBOX_URL in heroku config and the url is printed out fine but when I try to run the rake command 
heroku run rake searchkick:reindex CLASS=Model
I get a rake aborted error saying invalidURIerror

**Comment by ankane** [comment]
It sounds like you haven't successfully set `ELASTICSEARCH_URL`. Try running:

```sh
heroku config:set ELASTICSEARCH_URL=PASTE_VALUE_OF_SEARCHBOX_URL_HERE
```

**Comment by WilfredKesh** [comment]
Thanks man, I hadn't even checked the config value of ELASTICSEARCH_URL
and its literally 
ELASTICSEARCH_URL=heroku config:get SEARCHBOX_URL. 

When I got the "config:get is invalid, must be in the format Foo=bar" error, I changed the `` in this script :

heroku config:set ELASTICSEARCH_URL=`heroku config:get SEARCHBOX_URL`

 to double quotes and thought it worked but all it did was turn the function call into a string. Still do not know why it wont run config:get but I had to add the URL manually to work.
